### DISCUSSION TITLE
ciw, diw, viw equivalents in foreign language (Russian) doesn't work.

### DISCUSSION THREAD

**Discussion by iamyegor** [VSCodeVim/Vim#8613] [Q&A]
I can't use the diw, ciw viw equivalents in my primary language. They only work when I'm on the English layout. Other keybindings like dw, cw work just fine. This is my vscode .vimrc file that maps the English layout to Russian:
set encoding=utf-8
`nmap j gj
nmap k gk
nmap ; $
nmap Ж :
nmap й q
map й q
map ц w
map у e
map к r
map е t
map н y
map г u
map ш i
map щ o
map з p
map х [
map ъ ]
map ф a
map ы s
map в d
map а f
map п g
map р h
map о j
map л k
map д l
map ж ;
map э '
map ё \
map я z
map ч x
map с c
map м v
map и b
map т n
map ь m
map б ,
map ю .
map Й Q
map Ц W
map У E
map К R
map Е T
map Н Y
map Г U
map Ш I
map Щ O
map З P
map Х {
map Ъ }
map Ф A
map Ы S
map В D
map А F
map П G
map О J
map Л K
map Ж :
map Э "
map Я Z
map Ч X
map С C
map М V
map И B
map Т N
map Ь M
map Б <
map Ю >
map Ё /|`

**Comment by Edgar-P-yan** [comment]
It works in settings.json if you map the whole chord:
```
    { "before": ["с"], "after": ["c"] },
    { "before": ["с", "ш", "ц"], "after": ["c", "i", "w"] },
```

**Reply by iamyegor** [reply]
Wow, incredible! I finally received an answer after 1.5 years. A whole lot of things have happened since then. Thank you for the help, anyway
